server/gitlab/api.go

```diff
@@ -611,9 +611,10 @@ func (g *gitlab) GetUnreads(ctx context.Context, user *UserInfo, token *oauth2.T
 		}
 		opt.Page = resp.NextPage
 	}
+
 	notifications := make([]*internGitlab.Todo, 0, len(todos))
 	for _, todo := range todos {
-		if g.checkGroup(strings.TrimSuffix(todo.Project.PathWithNamespace, "/"+todo.Project.Path)) != nil {
+		if todo.Project != nil && g.checkGroup(strings.TrimSuffix(todo.Project.PathWithNamespace, "/"+todo.Project.Path)) != nil {
```

Comment:
Might as well be extra careful and nil check the todo as well, since we don't actually know which one was nil in the original issue

```suggestion
		if todo != nil && todo.Project != nil && g.checkGroup(strings.TrimSuffix(todo.Project.PathWithNamespace, "/"+todo.Project.Path)) != nil {
```